src/main/java/utils/SwipeHelper.java

```diff
@@ -0,0 +1,155 @@
+package utils;
+
+import com.google.common.collect.ImmutableList;
+import io.appium.java_client.*;
+import io.appium.java_client.android.AndroidDriver;
+import io.appium.java_client.ios.IOSDriver;
+import io.appium.java_client.touch.WaitOptions;
+import io.appium.java_client.touch.offset.PointOption;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.Point;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Pause;
+import org.openqa.selenium.interactions.PointerInput;
+import org.openqa.selenium.interactions.Sequence;
+
+import java.time.Duration;
+import java.util.HashMap;
+
+public class SwipeHelper {
+
+    private static Dimension windowSize;
+    private static int ANDROID_SCROLL_DIVISOR = 3;
+    private static Duration SCROLL_DUR = Duration.ofMillis(1000);
```

Comment:
ALL constants go to variables, probably in a config file

Reply:
At the momento, I put that constant in .env file